src/shared/otterizecloud/otterizecloudclient/cloud_client.go

```diff
@@ -33,9 +38,28 @@ func NewClient(ctx context.Context) (graphql.Client, bool, error) {
 		AuthStyle:    oauth2.AuthStyleInParams,
 	}
 
+	rootCAs, err := x509.SystemCertPool()
+	if err != nil {
+		return nil, false, fmt.Errorf("error loading root system cert pool: %w", err)
+	}
+
+	for _, path := range extraCAPEMPaths {
+		logrus.Infof("Loading root CA from cert PEM file at '%s'", path)
+		cert, err := os.ReadFile(path)
+		if err != nil {
+			logrus.Errorf("Error loading cert PEM file at '%s', trying to continue without it: %s", path, err)
```

Comment:
Should skip this iteration if you get an error, or the next function call will fail